### PR TITLE
Full Re-Design

### PR DESCRIPTION
**Pull request by kerrtravers**
### Why are you making this request?
The site looked a bit old and plain, So I've taken the css from my old site and customized it to fit the style of Smelt.

### What issue does this reference/resolve?
#3

### Preview 
capture

### Tested on:
http://vutondesign.com/smelt-modding.github.io/

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -1,13 +1,94 @@
-html {
-	background:black;
-	color:white;
-	font-family:Courier New, monospace;
+* {
```
Comment (Kenny2github) [inline stylesheet.css:1]: Nice, thanks for this

File: index.html [hunk 2]
```diff
@@ -17,65 +22,85 @@
...
+					<div class="boxes">
+
 		<h3 id="uninstall-part">To uninstall a command installed with SBPM:</h3>
 		<ol>
 			<li>Open Command Prompt (Win+R, type <code>cmd</code> and hit Enter/Win+X, C)</li>
 			<li>Type <code>sbpm uninstall &lt;package></code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
 			<li>You're done! A log will be available at <code>logs\uninstall.log</code>.</li>
 		</ol>
+			</div>
+					<div class="boxes">
```
Comment (Kenny2github) [inline index.html]: This is some weird indentation - would you mind making everything consistent? I.e.
```html
    </ol>
</div>
<div class="boxes">
```

File: index.html [hunk 3]
```diff
@@ -6,7 +6,12 @@
 		<title>Smelt: Batch Package Manager</title>
 	</head>
 	<body>
+		<center><img src="logo.png"><br/>
 		<h1>Smelt: Batch Package Manager</h1>
+	</center>
```
Comment (Kenny2github) [inline index.html]: Here, too - your indentation generally needs work.

**Review by Kenny2github — CHANGES_REQUESTED** [review]
Looks great! Though your design is on point (and judging from the screenshots, it’s looking a lot better), your code style needs a bit of work. Would you mind making consistent indentation? Thanks.

**Comment by kerrtravers** [comment]
@Kenny2github I'll work on all of this 👍

**Comment by kerrtravers** [comment]
Done, How's that?

**Review by Kenny2github — APPROVED** [review]
Looks better! I’m not going to be too nit picky about indentation (I can solve it myself) so I’ll merge this now.

**Pull request merged**